tests/suites/verify.js

```diff
@@ -135,6 +135,14 @@ export function verifySuite({
             verifier
           });
         });
+        it('If proofConfig.created is set and the value is not a valid ' +
+        '[XMLSCHEMA11-2] datetime, an INVALID_PROOF_DATETIME error MUST be ' +
+        'raised.', async function() {
```

Comment:
I think this no longer matches wording of spec, which I think now has consistent wording across all such statements, e.g., `if {condition}, an error MUST be raised which SHOULD communicate {error_type}`.

Reply:
you are correct the spec text has changed to: `If proofConfig.created is set and if the value is not a valid [[XMLSCHEMA11-2](https://w3c.github.io/vc-di-bbs/#bib-xmlschema11-2)] datetime, an error MUST be raised and SHOULD convey an error type of [PROOF_GENERATION_ERROR](https://www.w3.org/TR/vc-data-integrity/#PROOF_GENERATION_ERROR).`

Reply:
@TallTed thanks for the catch and updated here: https://github.com/w3c/vc-di-bbs-test-suite/pull/33/commits/46e767307e12482bea5ada24d2f0d16e73bd011f